bin/radical-pilot-create-static-ve

```diff
@@ -51,7 +51,7 @@ if ! test -z "$DEFAULTS"
 then
     # by default, install all RCT dependencies
     MODULES="$MODULES apache-libcloud chardet colorama idna msgpack"
-    MODULES="$MODULES msgpack-python netifaces ntplib parse pymongo<4"
+    MODULES="$MODULES msgpack-python netifaces ntplib parse dill"
     MODULES="$MODULES pyzmq regex requests setproctitle urllib3"
```

Comment:
@AymenFJA , @mtitov : should we add `mpi4py` as Mikhail suggested?  How do we handle failures when installing it?

Reply:
Mikhail and I discussed this and determined that mpi4py should not be added by default as to not make the bootstrapping instable (mpi4py does not always build cleanly).  If needed by a raptor run, the application would need to use `pilot.prepare_env` to create a raptor environment with the required modules.